client/signup/steps/woocommerce-install/transfer/transfer-site.tsx

```diff
@@ -56,7 +56,9 @@ export default function TransferSite( {
 		if ( ! siteId ) {
 			return;
 		}
-		dispatch( initiateAtomicTransfer( siteId, { softwareSet: 'woo-on-plans' } ) );
+		dispatch(
+			initiateAtomicTransfer( siteId, { softwareSet: 'woo-on-plans', context: 'woo-on-plans' } )
```

Comment:
All the other contexts use underscores so I was thinking maybe this one should too?

```suggestion
			initiateAtomicTransfer( siteId, { softwareSet: 'woo-on-plans', context: 'woo_on_plans' } )
```

Reply:
The problem is that the software name is `woo-on-plans` so I was trying to be consistent with that. It will be easier for the person doing the funnels to identify and not worry about casing.